frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -63,7 +63,9 @@ export default class ProfileLink extends Component {
       {
         title: t`Help`,
         icon: null,
-        link: MetabaseSettings.docsUrl(),
+        link: MetabaseSettings.isPaidPlan()
+          ? "https://www.metabase.com/help-premium/"
+          : "https://www.metabase.com/help/",
```

Comment:
I was tempted to move this inside MetabaseSettings, so we could do something like `MetabaseSettings.helpUrl()`. But I'm not sure if we'll do this in a lot of places, and I don't want to pollute the Settings file yet.

Reply:
I just updated it.